### PR TITLE
(fix bug). Fix all problem which Tester said.

### PR DESCRIPTION
**Pull request by AndriY2389**

### CODE REVIEW

File: src/app/list-offers/list-offers.component.ts [hunk 1]
```diff
@@ -92,6 +93,9 @@ export class ListOffersComponent implements OnInit {
     await new Promise(resolve => setTimeout(() => resolve(), ms)).then(() => this.getOfferDTOs());
   }
 
+  sortFn(n1: OfferDTO , n2: OfferDTO) {
+    return (n2.id - n1.id);
```
Comment (SilichAlex) [inline src/app/list-offers/list-offers.component.ts]: n1, n2?

File: src/app/list-contracts/list-contracts.component.ts [hunk 2]
```diff
@@ -138,6 +140,7 @@ export class ListContractsComponent implements OnInit {
             return (item.id !== x.id);
           });
           this.orderDTOs.push(x);
```
Comment (SilichAlex) [inline src/app/list-contracts/list-contracts.component.ts:142]: x?

File: src/app/list-contracts/list-contracts.component.ts [hunk 3]
```diff
@@ -50,6 +50,7 @@ export class ListContractsComponent implements OnInit {
     this.listOrderService.getAllOrders()
       .subscribe((x) => {
```
Comment (SilichAlex) [inline src/app/list-contracts/list-contracts.component.ts:51]: x?

File: src/app/create-contract/create-contract.component.ts [hunk 4]
```diff
@@ -113,6 +114,7 @@ export class CreateContractComponent implements OnInit {
     this.createOrderService.getCustomerByUserId(id)
       .subscribe((x) => {
```
Comment (SilichAlex) [inline src/app/create-contract/create-contract.component.ts:115]: x?

**Review by SilichAlex — APPROVED** [review]

**Review by AnastasiiaHanuliak — APPROVED** [review]

**Pull request merged**